rs/tests/consensus/tecdsa/BUILD.bazel

```diff
@@ -346,3 +343,45 @@ system_test_nns(
         "@crate_index//:tokio",
     ],
 )
+
+system_test_nns(
+    name = "tecdsa_performance_test",
+    additional_colocate_tags = [
+        "system_test_benchmark",
+    ],
+    env = {
+        "TECDSA_PERFORMANCE_TEST_KEY_IDS": "ecdsa_secp256k1",
+        "BENCHMARK_NAME": "tecdsa_performance_test",
+    },
+    tags = ["manual"],
+    test_driver_target = tecdsa_performance_test_template.test_driver_target,
+    runtime_deps = GUESTOS_RUNTIME_DEPS + GRAFANA_RUNTIME_DEPS,
+)
+
+system_test_nns(
+    name = "tschnorr_ed25519_performance_test",
+    additional_colocate_tags = [
+        "system_test_benchmark",
+    ],
+    env = {
+        "TECDSA_PERFORMANCE_TEST_KEY_IDS": "schnorr_ed25519",
+        "BENCHMARK_NAME": "tschnorr_ed25519_performance_test",
+    },
+    tags = ["manual"],
+    test_driver_target = tecdsa_performance_test_template.test_driver_target,
+    runtime_deps = GUESTOS_RUNTIME_DEPS + GRAFANA_RUNTIME_DEPS,
+)
+
+system_test_nns(
+    name = "tschnorr_bip340_performance_test",
+    additional_colocate_tags = [
+        "system_test_benchmark",
+    ],
+    env = {
+        "TECDSA_PERFORMANCE_TEST_KEY_IDS": "schnorr_bip340",
+        "BENCHMARK_NAME": "tschnorr_ed25519_performance_test",
+    },
+    tags = ["manual"],
+    test_driver_target = tecdsa_performance_test_template.test_driver_target,
+    runtime_deps = GUESTOS_RUNTIME_DEPS + GRAFANA_RUNTIME_DEPS,
+)
```

Comment:
Don't these performance system-tests also require the `colocated_test_driver_vm_required_host_features` and `colocated_test_driver_vm_resources` settings like you have in the template above? 

You probably also want to tag them with `experimental_system_test_colocation` as well.